xmtp/src/vmac_traits.rs

```diff
@@ -0,0 +1,60 @@
+use xmtp_proto::xmtp::v3::message_contents::vmac_unsigned_public_key::{
```

Comment:
I find this file naming interesting as there appears to be no traits in it, only Impls. 😄

Reply:
I can rename to proto_helpers or move code back to vmac.rs? I have no strong opinions here